amun/base/openshift-templates/inspection-workflow.yaml

```diff
@@ -115,20 +117,18 @@ objects:
           continueOn:
             failed: true
 
-parameters:
-- name: AMUN_INSPECTION_ID
-  description: Id of inspection that is run.
-  displayName: Inspection id
-  required: true
+- name: THOTH_FORCE_SYNC
+  description: should inspection complete message cause graph sync job to be run with force sync
+  displayName: force sync
```

Comment:
These changes cant go in, as it would disrupt other environments.
i noticed this is already been add to ocp4 and test, so kindly revert the changes in file from base.

Reply:
Ah yes, sorry. Thanks for catching this Harshad